src/identifiers.md

```diff
@@ -21,5 +23,7 @@ Or
 * The identifier is more than one character. `_` alone is not an identifier.
 * The remaining characters are alphanumeric or `_`.
 
+A raw identifier is like a normal identifier, but prefixed by `r#`, which is not included as part of the identifier. Unlike a normal identifier, a raw identifier may be any strict or reserved keyword.
```

Comment:
"...any strict or reserved keyword" except `crate`, `extern`, `self`, `Self` or `super`.

Reply:
Okay, I thought there might be exceptions, thanks.